tests/unit/drivers/test_rankingevaluation_driver.py

```diff
@@ -170,3 +171,66 @@ def test_evaluate_assert_doc_groundtruth_structure(simple_chunk_evaluate_driver,
     simple_chunk_evaluate_driver.eval_request = eval_request_with_unmatching_struct
     with pytest.raises(AssertionError):
         simple_chunk_evaluate_driver()
+
+
+def _compute_dcg(gains):
+    from math import log
+    """Compute discounted cumulative gain."""
+    ret = 0.0
+    for score, position in zip(gains[1:], range(2, len(gains) + 1)):
+        ret += score / log(position, 2)
+    return gains[0] + ret
+
+
+def _compute_idcg(gains):
+    """Compute ideal discounted cumulative gain."""
+    sorted_gains = sorted(gains, reverse=True)
+    return _compute_dcg(sorted_gains)
```

Comment:
Why do we have NDCG logic in the test?

Reply:
just for a test, I would like to have something more stupid there, can u think of some other stupid measure we can take based on scores? I also do not like having this logic here.

Reply:
I think precision_at_k is the dumbest and can be taken from [here](https://gist.github.com/bwhite/3726239)

Reply:
The point is that precision is based on ids of matches while ndcg works on relevancy scores. That is the difference I want to test, but anyway is an unnecesssry test. Will close